packages/@angular/cli/models/webpack-configs/typescript.ts

```diff
@@ -14,6 +14,8 @@ const webpackLoader: string = g['angularCliIsLocal']
 
 function _createAotPlugin(wco: WebpackConfigOptions, options: any) {
   const { appConfig, projectRoot, buildOptions } = wco;
+  options.compilerOptions = options.compilerOptions || {};
+  options.compilerOptions.preserveSymlinks = wco.buildOptions.preserveSymlinks;
```

Comment:
I'd rather have 

```
if (wco.buildOptions.preserveSymlinks) {
  options.compilerOptions.preserveSymlinks = true;
}
```

Would that prevent the flag from working as intended?

Reply:
(the subtle difference is if `--preserve-symlinks` is not passed on the command line we do not override the configuration the user set for himself)

Reply:
Agree, that's better. Fixed.